OracleDatabase/SingleInstance/samples/kubernetes/README.md

```diff
@@ -0,0 +1,161 @@
+Example of Kubernetes deployment
+===========================================================
+
+This example will describe how to deploy the Oracle Database image using native Kubernetes objects. 
+It's not intended for production use as the values used are not necessarily appropriate. 
+
+# Configuration
+This example YAML file will create:
+* A namespace _example-namespace_ to hold all objects
+* A ConfigMap _oracle-rdbms-config_ that describes database initiization settings (characterset, name, etc)
+* A PersistantVolumeClaim _oracle-rdbms-oradata_ that will hold the database files
+* A Deployment _oracle-rdbms_ that starts a container with the settings from _oracle-rdbms-config_ using the 
+    volume _oracle-rdbms-oradata_ and ensures it remains running
+* A Service _database_ that directs traffic flow into the database container
+
+Some key values customizable values:
+* _oracle-rdbms_
+  * image: "local-repo.com/oracle/database:19.3.0-ee"
+  * livenessProbe.initialDelaySeconds: 300        # In seconds, this is five minutes
+* _oracle-rdbms-config_
+  * ORACLE_CHARACTERSET: "AL32UTF8"
+  * ORACLE_EDITION: "enterprise"
+  * ORACLE_SID: "ORCLCDB"
+  * ORACLE_PDB: "ORCLPDB1"
+* _oracle-rdbms-oradata_
+  * storage: 10Gb
+
+You **must** change the image path to match your local image registry. 
+
+The livenessProbe will restart a container if it is not ready within the timeout, development systems may 
+    require more time and this should be adjusted accordingly
+
+# Deployment
+
+1. Deploy the objects
+    ```
+    $ kubectl apply -f kubernetes-example.yaml
+    namespace/example-namespace created
+    persistentvolumeclaim/oracle-rdbms-oradata created
+    configmap/oracle-rdbms-config created
+    deployment.apps/oracle-rdbms created
+    service/database created   
+    ```
+
+1. Create a password
+
+   Define your own password or generate one as in the example
+    ```
+    $ PASS=$(head /dev/urandom | tr -dc [:alnum:][:graph:] | head -c15) 
```

Comment:
CONTRIBUTING.md suggested I used `openssl rand` to generate a password but I wanted additional complexity (special characters). Happy to change if you think it complicates cross platform compatibility